scss/components/_list-styles.scss

```diff
@@ -42,6 +43,11 @@
   }
 }
 
+// Alternate list styles
+ul.row {
```

Comment:
Where is this used? We already have a `.row` class. If it's not the same, we should make a new class name for it.